spyplane/listeners.py

```diff
@@ -19,57 +20,63 @@ def __init__(self):
         pass
 
 
+post_service = PostAfterTickService()
+
+
 @bot.event
 async def on_ready():
-    print(f'{bot.user.name} has connected to Discord server. Version: {__version__}')
+    log(f'{bot.user.name} has connected to Discord server. Version: {__version__}')
     bot.channel = bot.get_channel(CONTROL_CHANNEL)
     bot.lock = asyncio.Lock()
     emoji = bot.get_emoji(EMOJI_BULLSEYE)
     bot.emoji_bullseye = emoji or '✅'
-    await bot.channel.send(f'{bot.user.name} has connected to Discord server. Version: {__version__}')
+
+    # Cron in not needed anymore, we are able to read embeds, and BGS Bot messages can trigger spy plane.
+    # await bot.channel.send(f'{bot.user.name} has connected to Discord server. Version: {__version__}')
+    # @aiocron.crontab('0/10 * * * *')
+    # async def tick_cron_job():
+    #     await post_service.tick_check_and_schedule()
 
```

Comment:
Cron-based tick-check was developed, and commented out. 
Reason: The BGSbot-based tick-check worked correctly, once the Intents were set correctly